docs/supported/index.rst

```diff
@@ -0,0 +1,267 @@
+========================
+Supported Configurations
+========================
+
+Supported Cluster Managers
+==========================
+
+.. raw:: html
+
+	<div class="pods-eq">
+	    <div class="pod-boxout pod-boxout--orchestration pod-boxout--recommended">
+			<img src="/_images/docker2x.png" alt="Docker logo"/>
+			<span>Docker, Swarm, Compose <em>Fastest</em></span>
```

Comment:
Can you please consider removing ``Fastest``? It might mislead user to believing that Docker-Swarm-Compose stack performs better than Kubernetes/Mesos. Thanks!